examples/advanced/case-studies/basic-forms/index.xml.njk

```diff
@@ -1,5 +1,5 @@
 ---
-permalink: "/case-studies/basic-forms/index.xml"
+permalink: "/advanced/case-studies/basic-forms/index.xml"
```

Comment:
given that we now have a middleware that generates the permalink, should we plan to remove them all from every files?

Reply:
yes, will handle this in another PR